### ISSUE TITLE
Undefined name: 'signer_email' in docs/gke/iap_request.py

### ISSUE THREAD

**Issue by cclauss** [kubeflow/kubeflow#2229]
__signer_email__ is [defined on line 40](https://github.com/kubeflow/kubeflow/blob/master/docs/gke/iap_request.py#L40) but not at a global scope so it is an [_undefined name_ on line 140](https://github.com/kubeflow/kubeflow/blob/master/docs/gke/iap_request.py#L140) with the potential to raise NameError at runtime.

[flake8](http://flake8.pycqa.org) testing of https://github.com/kubeflow/kubeflow on Python 3.7.1

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./docs/gke/iap_request.py:140:25: F821 undefined name 'signer_email'
                        signer_email))
                        ^
1    F821 undefined name 'signer_email'
1
```
__E901,E999,F821,F822,F823__ are the "_showstopper_" [flake8](http://flake8.pycqa.org) issues that can halt the runtime with a SyntaxError, NameError, etc. Most other flake8 issues are merely "style violations" -- useful for readability but they do not effect runtime safety.
* F821: undefined name `name`
* F822: undefined name `name` in `__all__`
* F823: local variable name referenced before assignment
* E901: SyntaxError or IndentationError
* E999: SyntaxError -- failed to compile a file into an Abstract Syntax Tree

**Comment by jlewi** [comment]
Anyone interested in providing a PR?

**Comment by cchung100m** [comment]
Hi @jlewi , @cclauss ,

I'm interested in working on this if no one else has signed up. However this is the first time I'm contributing to kubeflow so I may need some hand-holding,

Many thanks,

**Comment by cclauss** [comment]
You might review the proposed fix in #2283

**Comment by cchung100m** [comment]
Hi @cclauss ,
Thanks for the suggestion! :)
